src/main/java/mate/jdbc/service/CarServiceImpl.java

```diff
@@ -0,0 +1,56 @@
+package mate.jdbc.service;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+import mate.jdbc.dao.CarDao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+
+@Service
+public class CarServiceImpl implements CarService {
+    @Inject
+    private CarDao carDao;
+
+    @Override
+    public Car create(Car car) {
+        return carDao.create(car);
+    }
+
+    @Override
+    public Car get(Long id) {
+        return carDao.get(id).orElseThrow(() -> new NoSuchElementException("Could not get car "
```

Comment:
```suggestion
        return carDao.get(id).orElseThrow(() -> new DataProcessingException("Could not get car "
```
You could throw your custom exception here

Reply:
I used the template from other implemented services given